### DISCUSSION TITLE
import Axis

### DISCUSSION THREAD

**Discussion by Altlogik** [pybricks/support#1164] [General]
Good afternoon
When I enter
from pybricks.parameters import Axis

get the answer
from pybricks.parameters import Axis

what to do?

**Comment by BertLindeman** [comment]
You might have given a bit more info about what you do with what firmware and on what pybricks site.

Just guessing:
If you are using code.pybricks.com then axis would be there: [pybricks.geometry.Axis](https://code.pybricks.com/static/docs/v2.7.0/geometry.html?highlight=axis#pybricks.geometry.Axis)

In general if you are using beta.pybricks.com or code.pybricks.com the doc is available on the source window with the small book icon.

If you need more help, please add some more info.

Bert

**Reply by Altlogik** [reply]
I took an example from here
https://docs.pybricks.com/en/latest/hubs/primehub.html

**Reply by Altlogik** [reply]
![image](https://github.com/pybricks/support/assets/139757577/6b6adcdd-c08d-477f-aebb-4a238dcd52c7)

**Reply by BertLindeman** [reply]
OK,

When I run this program on `https://beta.pybricks.com/` and on recent firmware, this works fine. 
```python
from pybricks.hubs import PrimeHub
from pybricks.tools import wait
from pybricks.parameters import Axis

# Initialize the hub.
hub = PrimeHub()

# Get the acceleration or angular_velocity along a single axis.
# If you need only one value, this is more memory efficient.
while True:

    # Read the forward acceleration.
    forward_acceleration = hub.imu.acceleration(Axis.X)

    # Read the yaw rate.
    yaw_rate = hub.imu.angular_velocity(Axis.Z)

    # Print the yaw rate.
    print(yaw_rate)
    wait(100)
```

An example of the output:
```
0.03635502
0.03635502
-0.03364503
0.1063551
0.03635502
-0.1036628
```

To see the firmware you might run this small program:

```python
from pybricks import version
print(version)
```

For me this shows:
```
('primehub', '3.3.0b8', 'v1.20.0-23-g6c633a8dd on 2023-07-07')
```

What does it show for you?

Bert

**Comment by Altlogik** [comment]
OK
I will try https://beta.pybricks.com/
